### DISCUSSION TITLE
Maps settings inconsistent across devices

### DISCUSSION THREAD

**Discussion by freeasabeer** [devemux86/cruiser#568] [Q&A] [answered]
Hello,
I don't know what's happenning on my phones, but here are the maps settings I have on 3 different devices

1. on the tablet (I believe here the settings are the expected ones)
![Tablet](https://github.com/devemux86/cruiser/assets/149788/b4b0d7c3-a7cf-4c15-a36e-321221618e0e)

2. on phone 1 (some maps settings are missing)
![Phone#1](https://github.com/devemux86/cruiser/assets/149788/666b3f72-d185-4b0c-bc75-22fa0ba20e92)

3. on phone 2 (even more maps settings are missing)
![Phone#2](https://github.com/devemux86/cruiser/assets/149788/12dfebed-b9f1-4302-abb5-4f0d835fee9f)

What can be wrong on my devices ?

**Comment by devemux86** [comment, marked as answer]
Hi @freeasabeer,

The menu depends on the opened map and if the "Extended menu" is enabled in "Settings | Application | Advanced".

1. You use an offline map and enabled the "Extended menu".

2. You use an offline map and **not** enabled the "Extended menu".

3. You use an online map and **not** enabled the "Extended menu".

By the way, if you do not open external maps / themes, you do not need the "Extended menu".
You can leave it disabled, so the app menus are not very complicated.

**Comment by freeasabeer** [comment]
Ahhhh, I got it now !
Many thanks to have clarified the way it works.
Side note: I really thought I had the same settings on all devices which actually was not the case...
